array-map/main.js

```diff
@@ -0,0 +1,20 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+const doubled = numbers.map(number => number * 2);
+console.log('doubled', doubled);
+
+const prices = numbers.map(number => `$${number}.00`);
```

Comment:
While this works in this specific instance because there are only whole numbers, what would happen if the numbers had decimals already? Look into using the `toFixed` method:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
![image](https://user-images.githubusercontent.com/71291742/135144114-85f7f77d-6e6d-48a6-bfc0-77370857941f.png)